projects/angular/src/forms/select/STYLES.md

```diff
@@ -0,0 +1,23 @@
+# Select - Style API
+
+## CSS Custom Properties
+
+| CSS Custom Property                          | Description                                       |
+| -------------------------------------------- | ------------------------------------------------- |
+| --clr-forms-select-option-color              | Text color used in the options of a single select |
+| --clr-forms-select-multiple-background-color | Background color used in a multiple select        |
+| --clr-forms-select-multiple-border-color     | Border color used by multiple select options      |
+| --clr-forms-select-multiple-option-color     | Text color used by the multiple select options    |
+
+## Class names
+
+| Class name              | Description                                                               |
+| ----------------------- | ------------------------------------------------------------------------- |
+| clr-select              | Select element class                                                      |
+| clr-control-helper      | Helper element class when clr-select-container component is used          |
+| clr-control-error       | Error element class when clr-select-container component is used           |
+| clr-control-success     | Success element class when used with clr-select-container component       |
+| clr-validate-icon       | Error or success icon class when used with clr-select-container component |
+| clr-control-container   | Internal class that styles the clr-select-container component             |
```

Comment:
These are common for all the forms elements and they come from the container. I don't think we should be documenting them in each and every form element.

Reply:
Update as requested